assets/scss/templates/_integrations.scss

```diff
@@ -0,0 +1,83 @@
+body {
+  &[data-active-menu="13"] {
+    background-color: #f6f6f6
```

Comment:
I think that we don't have this in variable, but it would be nice to have. Could you add this please?

Reply:
Sure! 🙌